### PR TITLE
Align Nav2 Config with cp3zero config

### PR DESCRIPTION
**Pull request by bgill92**
Initial parameters for nav2

### CODE REVIEW

File: c3pzero_navigation/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# c3pzero_navigation
```
Comment (griswaldbrooks) [inline c3pzero_navigation/README.md:1]: Can you link this in the main README as in https://github.com/PickNikRobotics/c3pzero_siteconfig/pull/29/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

File: c3pzero_navigation/params/nav2_params.yaml [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+
+bt_navigator:
+  ros__parameters:
+    use_sim_time: True
+    global_frame: map
+    robot_base_frame: base_link
+    odom_topic: /odom
+    bt_loop_duration: 10
+    default_server_timeout: 20
+    # 'default_nav_through_poses_bt_xml' and 'default_nav_to_pose_bt_xml' are use defaults:
```
Comment (griswaldbrooks) [inline c3pzero_navigation/params/nav2_params.yaml]: Delete commented out params

File: c3pzero_navigation/params/nav2_params.yaml [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+      update_frequency: 5.0
+      publish_frequency: 2.0
+      global_frame: odom
+      robot_base_frame: base_link
+      use_sim_time: True
+      rolling_window: true
+      width: 3
+      height: 3
+      resolution: 0.05
+      # Can erither use the robot_radius or footprint parameter
```
Comment (griswaldbrooks) [inline c3pzero_navigation/params/nav2_params.yaml]: Typo

**Review by griswaldbrooks — CHANGES_REQUESTED** [review]

File: c3pzero_navigation/README.md (hunk 1 above)
Comment (bgill92) [reply c3pzero_navigation/README.md:1]: Done

File: c3pzero_navigation/params/nav2_params.yaml (hunk 2 above)
Comment (bgill92) [reply c3pzero_navigation/params/nav2_params.yaml]: Done

File: c3pzero_navigation/params/nav2_params.yaml (hunk 3 above)
Comment (bgill92) [reply c3pzero_navigation/params/nav2_params.yaml]: Changed

**Review by griswaldbrooks — APPROVED** [review]

**Review by abake48 — APPROVED** [review]
lgtm

**Pull request merged**